sonar-python-plugin/src/main/java/org/sonar/plugins/python/PythonPlugin.java

```diff
@@ -81,7 +84,12 @@ public void define(Context context) {
       addFlake8Extensions(context);
     }
     if (sonarRuntime.getProduct() == SonarProduct.SONARLINT) {
-      context.addExtension(SonarLintPythonIndexer.class);
+      try {
```

Comment:
Can we wrap this logic in a function like `isSonarLintPluginAPIProvided()` (or something similar)?